src/js/pages/services/ServicesTab.js

```diff
@@ -0,0 +1,223 @@
+import React from 'react';
+import {StoreMixin} from 'mesosphere-shared-reactjs';
+
+import AlertPanel from '../../components/AlertPanel';
+import Config from '../../config/Config';
+import DCOSStore from '../../stores/DCOSStore';
+import FilterHeadline from '../../components/FilterHeadline';
+import SaveStateMixin from '../../mixins/SaveStateMixin';
+import Service from '../../structs/Service';
+import ServiceDetail from '../../components/ServiceDetail';
+import ServiceFilterTypes from '../../constants/ServiceFilterTypes';
+import ServiceSidebarFilters from '../../components/ServiceSidebarFilters';
+import ServicesTable from '../../components/ServicesTable';
+import ServiceTree from '../../structs/ServiceTree';
+import SidebarActions from '../../events/SidebarActions';
+import SidePanels from '../../components/SidePanels';
+
+var DEFAULT_FILTER_OPTIONS = {
+  filterHealth: null,
+  searchString: ''
+};
+
+let saveState_properties = Object.keys(DEFAULT_FILTER_OPTIONS);
+
+var ServicesTab = React.createClass({
```

Comment:
Hey @rcorral, sorry for the delayed response - didn't see this thread yesterday for some reason. 

The outer `ServicesPage` module remains, but all its content is split out into  `ServicesTab`.  That means that whenever a PR which changes to `ServicesPage` gets merged - and pretty much half the PRs into the `feature/unified-ui` branch touch it - the work of splitting `ServicesPage` has to be redone during the subsequent rebase.

It's worth noting that this is reflected in the git history `ServicesTab` is actually the result of a `git mv ServicesPage.js services/ServicesTab.js` - this means that `ServicesTab` is technically not the new component, rather `ServicesPage` is (the identical naming makes this confusing and git doesn't handle it perfectly). 

This caused a lot of overhead in a previous PR on a private repository, where I had updated the relevant classes to ES6 style, because it wasn't just a copy-paste job - things like initial component state had to be updated as well. This was painstaking and error-prone, as well as being rather annoying for everybody involved. 

`I don't understand the opposition for sticking with this style.`
Excuse the quote, but I wanted to address it directly because I seem to be having a lot of difficulty communicating one thing: I don't oppose the adoption of ES6 classes for components. I would like this component to be an ES6 class. I will convert it to an ES6 class in a future PR. 

I am opposing making such a change _in this PR_ because I want to avoid either me or any other developer getting stuck in a rebase loop as a result.